### PR TITLE
Zeppelin import support

### PR DESCRIPTION
**Pull request by indig**
Adds drag-and-drop support for Zeppelin import. 

Also, new Notebook import button to use instead of supporting a URL in the create notebook button.

### CODE REVIEW

File: polynote-server/src/main/scala/polynote/server/repository/NotebookRepository.scala [hunk 1]
```diff
@@ -103,7 +105,19 @@ trait FileBasedRepository extends NotebookRepository[IO] {
               }
             },
             content => {
-              writeString(extPath, content)
+              if (relativePath.endsWith(".json")) { // assume zeppelin
```
Comment (jeremyrsmith) [inline polynote-server/src/main/scala/polynote/server/repository/NotebookRepository.scala]: 😬

Comment (indig) [reply polynote-server/src/main/scala/polynote/server/repository/NotebookRepository.scala]: yeah... any better ideas? I figured this was better than a whole other code path just for zeppelin... :-\

Comment (jeremyrsmith) [reply polynote-server/src/main/scala/polynote/server/repository/NotebookRepository.scala]: Zeppelin is really all around not a good citizen (as much as I prefer it to Jupyter). At least make it `.zeppelin.json` or something, ya jerks

Comment (indig) [reply polynote-server/src/main/scala/polynote/server/repository/NotebookRepository.scala]: Sorry not code path but user-interaction-path: somehow we need to tell whether this is a zeppelin file. We could do it in the UI by having a separate button or something but I figured it was better to kind of guess here. Definitely open to other ideas if you have any!

**Pull request merged**